opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectEditionOverview.jsx

```diff
@@ -342,7 +357,23 @@ const StixDomainObjectEditionContainer = (props) => {
         >
           {({ submitForm, isSubmitting, setFieldValue, values }) => (
             <Form style={{ margin: '20px 0 20px 0' }}>
-              {'name' in stixDomainObject && (
+              {'result_name' in stixDomainObject ? (
+                <Field
+                  component={TextField}
+                  variant="standard"
+                  name="result_name"
+                  label={t('Result Name')}
+                  fullWidth={true}
+                  disabled={typesWithoutName.includes(
+                    stixDomainObject.entity_type,
+                  )}
+                  onFocus={handleChangeFocus}
+                  onSubmit={handleResultName}
+                  helperText={
+                    <SubscriptionFocus context={editContext} fieldName="result_name" />
+                  }
+                />
+              ) : (
```

Comment:
don't we need a test here for 'name' like it was before ? or was is unnecessary ?

Reply:
You're right, I didn't reproduce the case where that test is necessary, but I think it's better to add it the way it was before !